chapters/object_types/volumes.txt

```diff
@@ -14,15 +14,15 @@ create a new volume object of given subtype `subtype` use
 ANARIVolume anariNewVolume(ANARIDevice, const char *subtype);
 ....
 
-[[object_types_volume_scivis]]
-==== Scivis
+[[object_types_volume_tf1d]]
+==== TransferFunction1D
 
-Feature: `KHR_VOLUME_SCIVIS`
+Feature: `KHR_VOLUME_TF1D`
```

Comment:
I see that it will be long (like `KHR_SPATIAL_FIELD_STRUCTURED_REGULAR`), but we wanted to avoid abbreviations, so it should be `KHR_VOLUME_TRANSFER_FUNCTION1D`.

Reply:
Fixed in  002ebdf.